linux/LinuxProcess.c

```diff
@@ -504,6 +702,8 @@ long LinuxProcess_compare(const void* v1, const void* v2) {
    #endif
    case IO_PRIORITY:
       return LinuxProcess_effectiveIOPriority(p1) - LinuxProcess_effectiveIOPriority(p2);
+   case COMM:
+      return strcmp(LinuxProcess_getCommandStr((Process *)p1), LinuxProcess_getCommandStr((Process *)p2));
```

Comment:
`LinuxProcess_getCommandStr` only works for Linux processes, it is necessary to cast here and back in `LinuxProcess_getCommandStr()`?

Reply:
Not necessary, but it simply follows LinuxProcess_writeField.